index.html

```diff
@@ -72,6 +72,9 @@ <h1 class="title">HERE Maps on GitHub</h1>
 				client_secret : '918cbdf3a49ba292ff5e51e01e893ae0117c93ac'
 				},
 				repos = [
+					{id: 'mobility-on-demand-use-cases-ios',
+					type: 'Objective-C',
+					description:'HERE mobility on-demand demo written as an iOS application'},
```

Comment:
I suppose the `mobility-on-demand-use-cases` part of the name refers to all use-cases to be presented on GitHub regardless of the flavour of programming language they're developed in.

The description more accurately describes what is to be found within the specific repo.